javascript_exercise.js

```diff
@@ -1,5 +1,5 @@
 // The function below is not working properly - can you fix it?
 
-function area(height width) {
-    height * width;
+function area(height, width) {
+    return height * width;
 }
```

Comment:
You need to call the function for example like 
area(10,10); and then it will run!
Great work with this!